test/chainsaw/testmetrics/dynatrace-provider/mock-server.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mockserver
+spec:
+  ports:
+    - name: serviceport
+      port: 1080
+      protocol: TCP
+      targetPort: serviceport
+  selector:
+    app: mockserver
+  sessionAffinity: None
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: mockserver
+  name: mockserver
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mockserver
+  template:
+    metadata:
+      labels:
+        app: mockserver
+      name: mockserver
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - env:
+            - name: MOCKSERVER_LOG_LEVEL
+              value: INFO
+            - name: SERVER_PORT
+              value: "1080"
+          image: mockserver/mockserver:mockserver-5.13.0
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              cpu: 500m
+              memory: 128Mi
+            requests:
+              cpu: 10m
+              memory: 64Mi
+          livenessProbe:
+            failureThreshold: 10
+            initialDelaySeconds: 10
+            periodSeconds: 5
+            successThreshold: 1
+            tcpSocket:
+              port: serviceport
+            timeoutSeconds: 1
+          name: mockserver
+          ports:
+            - containerPort: 1080
+              name: serviceport
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 10
+            initialDelaySeconds: 2
+            periodSeconds: 2
+            successThreshold: 1
+            tcpSocket:
+              port: serviceport
+            timeoutSeconds: 1
+          volumeMounts:
+            - mountPath: /config
+              name: config-volume
+            - mountPath: /libs
+              name: libs-volume
+      terminationGracePeriodSeconds: 30
+      volumes:
+        - configMap:
+            defaultMode: 420
+            name: mockserver-config
+            optional: true
+          name: config-volume
+        - configMap:
+            defaultMode: 420
+            name: mockserver-config
+            optional: true
+          name: libs-volume
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: mockserver-config
+data:
+  initializerJson.json: |-
+    [
+      {
+        "httpRequest": {
+          "method" : "GET",
+          "path" : "/api/v2/metrics/query",
+          "queryStringParameters" : {
+            "metricSelector" : [ "query" ]
+          }
+          "headers" : {
+            "host" : [ "mockserver.chainsaw-epic-spaniel.svc.cluster.local:1080" ],
```

Comment:
as mentioned before please get rid of this header since it depend on the test run (line 103), otherwise very autogenerated run will fail since the host will change

Reply:
Oh! @RealAnna  In the previous comment you made, you only pointed the removal only the `"x-forwarded-by" : [ "MockServer_99f6f711-462f-4a60-9432-303a23ddd8ec" ],` from the header. Anyways, I have removed the` host ` as well. If there is further adjustment, I will be glad to work to it out